### DISCUSSION TITLE
Where to host files on Gannet?

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1514] [Q&A]
Hey, I'm looking to upload some files for my paper on Gannet for storage! Nothing crazy huge, just 200mb or so. Where in the file structure should I upload them (and how can I get there)?

I logged on to afcoyle@Gannet, but that doesn't seem like the best place to keep them publically accessible. Any advice?

**Comment by kubu4** [comment]
You have a dedicated, publicly accessible directory on Gannet already. Your `Sebastes` folder is world viewable here:

https://gannet.fish.washington.edu/Sebastes/

If you want to add folders/files to that directory, you can do so via `ssh` or `rsync` to the following location: `/volume2/web/Sebastes/`

Alternatively, you could login to the web interface for Gannet and use the File Station to upload files that way (can click and drag, or browser files on your computer for upload): https://gannet.fish.washington.edu:5001/

**Reply by aspencoyle** [reply]
Thanks a bunch, Sam! I tried to use the web interface, but my account doesn't have permission to use the File Station (or Download Station, or Cloud Sync). Would you be able to give those permissions to me?

**Reply by kubu4** [reply]
Sorry about that! Weird. Fixed! Try again.

**Reply by aspencoyle** [reply]
Uploaded and organized everything, thanks Sam!
